crates/ty_python_semantic/resources/mdtest/protocols.md

```diff
@@ -893,8 +893,10 @@ class LotsOfBindings(Protocol):
     match object():
         case l:  # error: [ambiguous-protocol-member]
             ...
+    # error: [ambiguous-protocol-member] "Consider adding an annotation, e.g. `m: int | str = ...`"
+    m = 1 if 1.2 > 3.4 else "a"
```

Comment:
Yes, unlikely to ever appear in practice. But a nice way to demonstrate how the previous `Type::literal_promotion_type` was different from `Type::promote_literals` (this only works with the latter).